contracts/CollateralTracker.sol

```diff
@@ -228,7 +228,7 @@ contract CollateralTracker is ERC20Minimal, Multicall {
     /// @notice Initialize a new collateral tracker for a specific token corresponding to the Panoptic Pool being created by the factory that called it.
     /// @dev The factory calls this function to start a new collateral tracking system for the incoming token at 'underlyingToken'.
     /// The factory will do this for each of the two tokens being tracked. Thus, the collateral tracking system does not track *both* tokens at once.
-    /// @param underlyingToken the address of the token to track collateral for corresponding to the Panoptic Pool being created by the factory calling this (as 'msg.sender').
```

Comment:
Please revert this, the current grammar is as intended. Without the "for", it is unclear that we are tracking collateral balances *specifically* of that token